styles/components/alert.module.scss

```diff
@@ -0,0 +1,123 @@
+.popUpWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(68, 68, 68, 0.5);
+    margin: 0;
+    padding: 0;
+    opacity: 0;
+    z-index: 50;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
+}
+
+.popUp {
+    box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
```

Comment:
You don't need to add browser-specific prefixes. We use an `autoprefixer` module which does it automatically.

Reply:
Cool